SampleApp-Source/sample_app/src/main/java/com/liveperson/sample/app/FragmentContainerActivity.java

```diff
@@ -110,7 +111,12 @@ private void initFragment() {
 
             Log.d(TAG, "initFragment. authCode = "+ authCode);
             Log.d(TAG, "initFragment. publicKey = "+ publicKey);
-            LPAuthenticationParams authParams = new LPAuthenticationParams();
+            LPAuthenticationParams authParams;
+            if (!TextUtils.isEmpty(SampleAppStorage.getInstance(this).getAppInstallId())) {
+                authParams = new LPAuthenticationParams(LPAuthenticationType.UN_AUTH);
```

Comment:
If I'm not wrong, an authenticated conversation can also have an engagement. So, setting it to UnAuth mode based on appInstallId might not be a good idea.

Reply:
A new method `getAuthType()` is created.